packages/web-app-files/src/components/SideBar/Shares/Collaborators/CollaboratorsEditOptions.vue

```diff
@@ -335,6 +381,34 @@ export default {
       return role.name === 'advancedRole'
     },
 
+    confirmCustomPermissions() {
+      this.$refs.customPermissionsDrop.hide()
+
+      this.selectedRole = this.advancedRole
+      this.publishChange()
+    },
+
+    cancelCustomPermissions() {
+      this.$refs.customPermissionsDrop.hide()
+      this.$refs.rolesDrop.show()
+    },
+
+    getRecipientsCustomPermissions() {
```

Comment:
Name of the method is misleading. It internally sets a value instead of returning one. Something like `resetCustomPermissions` is more appropriate I think.